packages/inventory/src/containers/ListView/index.tsx

```diff
@@ -0,0 +1,173 @@
+  service: OpenSRPService,
+  geoLevel: 0,
+};
+
+export type ServicePointsListTypes = ServicePointsListProps & RouteComponentProps;
+
+/** component that renders product catalogue */
+
+const ServicePointList = (props: ServicePointsListTypes) => {
+  const {
+    service,
+    LocationsByGeoLevel,
+    rootLocations,
+    columns,
+    fetchLocationsCreator,
+    fetchTreesCreator,
+    baseURL,
+  } = props;
+  const { broken, errorMessage, handleBrokenPage } = useHandleBrokenPage();
+  const [loadingJurisdictions, setLoadingJurisdictions] = useState<boolean>(
+    rootLocations.length === 0
+  );
+  const [loadingHierarchy, setLoadingHierarchy] = useState<boolean>(
+    LocationsByGeoLevel.length === 0
+  );
+
+  useEffect(() => {
+    loadJurisdictions(fetchLocationsCreator, baseURL, undefined, undefined, service)
+      .catch((err: Error) => handleBrokenPage(err))
+      .finally(() => setLoadingJurisdictions(false));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    if (rootLocations.length > 0) {
+      const promises = rootLocations
+        .map((location) => location.id.toString())
+        .map((rootId) => loadHierarchy(rootId, fetchTreesCreator, baseURL, undefined, service));
+      Promise.all(promises)
+        .catch((err: Error) => handleBrokenPage(err))
+        .finally(() => setLoadingHierarchy(false));
+    } else {
+      setLoadingHierarchy(false);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [JSON.stringify(rootLocations)]);
+
+  if (loadingHierarchy || loadingJurisdictions) {
+    return <ServicePointsLoading />;
+  }
+
+  if (broken) {
+    return <BrokenPage errorMessage={errorMessage} />;
+  }
+
+  const pageTitle = `${SERVICE_POINT_INVENTORY} (${LocationsByGeoLevel.length})`;
+  // add a key prop to the array data to be consumed by the table
+  const dataSource = LocationsByGeoLevel.map((location) => {
+    const locationToDisplay = {
+      key: `${TableColumnsNamespace}-${location.model.id}`,
+      type: '',
+      serviceName: location.model.label,
+      location: getNodePath(location),
+      servicePointId: location.model.id,
+    };
+    return locationToDisplay;
+  });
+
+  return (
+    <div className="content-section">
+      <Helmet>
+        <title>{pageTitle}</title>
+      </Helmet>
+      <PageHeader title={pageTitle} className="page-header"></PageHeader>
+      <Row className={'list-view'}>
+        <Col className={'main-content'}>
+          <div className="main-content__header">
+            <Link to={ADD_SERVICE_POINT}>
+              <Button type="primary" size="large" disabled>
+                {ADD_SERVICE_POINT}
+              </Button>
+            </Link>
+          </div>
+          <Table dataSource={dataSource} columns={columns}></Table>
+        </Col>
+      </Row>
+    </div>
+  );
+};
+
+ServicePointList.defaultProps = defaultProps;
+
+export { ServicePointList as ProductCatalogueList };
```

Comment:
why are we exporting ServicePointList as ProductCatalogueList ?

Reply:
yeah so wierd, I don't even know